diktat-gradle-plugin/build.gradle.kts

```diff
@@ -1,42 +1,19 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.getCurrentOperatingSystem
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     `java-gradle-plugin`
-    kotlin("jvm") version "1.8.10"
-    jacoco
+    id("org.cqfn.diktat.buildutils.kotlin-jvm-configuration")
+    id("org.cqfn.diktat.buildutils.code-quality-convention")
     id("pl.droidsonroids.jacoco.testkit") version "1.0.9"
     id("org.gradle.test-retry") version "1.5.2"
 }
 
-repositories {
-    flatDir {
-        // to use snapshot diktat without necessary installing
-        dirs("../diktat-common/target")
-        content {
-            includeGroup("org.cqfn.diktat")
-        }
-    }
-    mavenCentral()
-    mavenLocal {
-        // to use snapshot diktat
-        content {
-            includeGroup("org.cqfn.diktat")
-        }
-    }
-}
-
-// default value is needed for correct gradle loading in IDEA; actual value from maven is used during build
-// To debug gradle plugin, please set `diktatVersion` manually to the current maven project version.
-val ktlintVersion = project.properties.getOrDefault("ktlintVersion", "0.46.1") as String
-val diktatVersion = project.version.takeIf { it.toString() != Project.DEFAULT_VERSION } ?: "1.2.3"
-val junitVersion = project.properties.getOrDefault("junitVersion", "5.8.1") as String
-val jacocoVersion = project.properties.getOrDefault("jacocoVersion", "0.8.7") as String
 dependencies {
     implementation(kotlin("gradle-plugin-api"))
-    implementation("io.github.detekt.sarif4k:sarif4k:0.3.0")
+    implementation(libs.sarif4k.jvm)
```

Comment:
probably it should be a dependency of `diktatCommon`

Reply:
`ktlint` uses the old version of `sarif4k` (0.0.1).
versions should be compatible, but will postpone upgrading till project is migrated